### ISSUE TITLE
Remove the draft training videos

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1355]
I created some 20 draft instructional videos on FreeCEN2 usage one year ago. These very recorded zoom sessions and the videos are stored on my personal YouTube account. 
It is time that these were removed from my account and replaced on FreeCEN2 by access to the professional instructional videos

**Comment by DeniseColbert** [comment]
I was thinking this yesterday, too! The Coords had some suggested improvements, but nothing major. If we're happy to not let the perfect be the enemy of the good, I'll upload them into YouTube when I don't need the bandwidth for meetings etc.

**Comment by DeniseColbert** [comment]
Work in progress, links here: https://docs.google.com/document/d/1xAbAFG4usBxV5zYxDF2EnsFeKDiu8A7EFJQ74rrHcCA/edit?usp=sharing

(document is only accessible by those logged into a FreeUKGen account)

**Comment by PatReynolds** [comment]
@Captainkirkdawson to deploy.

**Comment by Captainkirkdawson** [comment]
@DeniseColbert currently they are hard coded ; would it be better to place them in a refinery page and place a link to that in the code; that way you or others can make the changes in future?

**Comment by Captainkirkdawson** [comment]
@DeniseColbert @PatReynolds there is one additional video for county coordinators on loading/uploading managing VLD files.

It also covers how to enter information on the transcriber of the VLD file

https://youtu.be/GUr_LbzNH0o

**Comment by DeniseColbert** [comment]
Thanks, Kirk. I have downloaded that one from your channel, and uploaded it to ours (as we don't have an equivalent). I have embedded all videos in this page on Refinery: https://test3.freecen.org.uk/cms/draft-instructional-videos. Next steps are to tidy it up and make it live.

**Comment by AlOneill** [comment]
I had a quick look at the first video and have the following comments:

* the views of the site pages are distorted
* it is not possible to read most of the red comments, even in full screen
* If you select "View on Youtube", the opening view (with the start button) is clipped at the sides
* the closed captions are pretty accurate but they render "FreeCEN" as "free send" and "freecen.org.uk" as "freesend.org.uk"

**Comment by PatReynolds** [comment]
@DeniseColbert to make page live.

**Comment by Captainkirkdawson** [comment]
@DeniseColbert The link has been made and it is active on production.
I noticed 2 things. Firstly one cannot view the pages in full screen which makes them very hard to follow.
Secondly there is no video on the login process and how to recover the password
![login video.png](https://images.zenhubusercontent.com/5b68ae684b5806bc2bdefcf0/b9838bf9-7d9a-415a-8c3f-bc81e4463b6d)
